packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -768,6 +768,26 @@ window.addEventListener("error", function(e) {
     await controller.addUserScript(overrideScript);
   }
 
+  @override
+  Future<void> setOverScrollMode(WebViewOverScrollMode mode) async {
+    switch (mode) {
+      case WebViewOverScrollMode.always:
+        return _webView.scrollView.setBounces(true);
```

Comment:
Don't you need to set the "always bounce" flags to true in this branch?

Reply:
Ah, good catch. I read the documentation again and yes these do also need to be set.